crates/goose-server/src/routes/recipe.rs

```diff
@@ -34,6 +35,26 @@ pub struct CreateRecipeResponse {
     error: Option<String>,
 }
 
+#[derive(Debug, Deserialize)]
```

Comment:
It looks like this entire recipe thing is not part of our open-api integration somehow. this should be integrated with openapi.rs and then we can autogenerate the ts code. can you check how much work it is to do this the right way? if too much, we can do in a follow up